pycobertura/cli.py

```diff
@@ -108,7 +122,7 @@ def show(
         filesystem=filesystem_factory(source, source_prefix=source_prefix),
     )
     Reporter = reporters[format]
-    reporter = Reporter(cobertura, ignore_regex)
+    reporter = Reporter(cobertura, ignore_regex, hide_columns)
```

Comment:
It feels like passing a comma-separated string to `Reporter` is a leaky abstraction from the command line interface. Can we have `Reporter` take a Python list of column names and deserialize the CLI input before passing it to `Reporter`?

Reply:
done in https://github.com/aconrad/pycobertura/pull/160/commits/805ceff720a025ed5e9247d0925c63dbf4c650d6